recipes/oslo.serialization/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "oslo.serialization" %}
+{% set version = "2.20.0" %}
+{% set sha256 = "fca6fbb350d560aab8a4fdc9a1128dac3b1d38b2fc9bf5ad22136ae090854802" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - pbr
+    - six
```

Comment:
FWICT `six` is not needed at build time. So would recommend dropping it from `build`.